website/utils/fetchUpdates.ts

```diff
@@ -0,0 +1,30 @@
+import fs from 'fs';
+import { join } from 'path';
+import matter from 'gray-matter';
+
+export interface UpdateProps {
+  date: string;
+  story: string;
+  route?: string;
+  updateURL?: string;
+}
+
+const updatesDirectory = join(process.cwd(), 'updates');
+
+function getUpdateSlugs() {
+  return fs.readdirSync(updatesDirectory);
+}
+
+function getIndividualUpdate(updateName) {
+  const slug = updateName.replace(/\.md$/, '');
+  const fullPath = join(updatesDirectory, `${slug}.md`);
+  const fileContents = fs.readFileSync(fullPath, 'utf8');
```

Comment:
would it be much trouble to use the async api?

Reply:
nope, that's definitely the right call